dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -59,7 +59,7 @@
       fallbackResponse: '#{@fallback_response.to_json}',
       callback: '#{@callback}',
       app: 'unplug',
-      level: '#{@level.level_num}',
+      level: '#{escape_javascript(@level.level_num)}',
```

Comment:
For my own understanding, is this primarily done to inhibit the warning, or is this usage of level actually more dangerous than the others, where there are comments about sourcing from static data?

Reply:
What was the answer? :)